boards/DCM/Src/App/torquevectoring/App_ActiveDifferential.c

```diff
@@ -0,0 +1,37 @@
+#include "torquevectoring/App_ActiveDifferential.h"
+#include "torquevectoring/App_TorqueVectoringConstants.h"
+#include <math.h>
+#include <stdlib.h>
+
+
+void App_ActiveDifferential_ComputeTorque(ActiveDifferential_Inputs *inputs, ActiveDifferential_Outputs *outputs) {
+    float Delta = App_ActiveDifferential_WheelAngleToSpeedDelta(inputs->wheel_angle_deg);
+    float cl = 1+Delta;
+    float cr = 1-Delta;
+
+    float torque_lim_Nm = App_ActiveDifferential_PowerToTorque(inputs->power_max_kW, inputs->motor_speed_left_rpm, inputs->motor_speed_right_rpm, cl, cr);
+
+    float torque_left_Nm = torque_lim_Nm * (1 + Delta);
+    float torque_right_Nm = torque_lim_Nm * (1 - Delta);
+    float torque_max_Nm = fmaxf(torque_left_Nm, torque_right_Nm);
+
+    float scale = 1.0f;
+    if (torque_max_Nm > MOTOR_TORQUE_LIMIT_Nm) {
+        scale = MOTOR_TORQUE_LIMIT_Nm / torque_max_Nm;
+    }
+
+    outputs->torque_left_Nm = torque_left_Nm * scale;
+    outputs->torque_right_Nm = torque_right_Nm * scale;
+}
+
+float App_ActiveDifferential_SteeringAngleToSpeedDelta(float wheel_angle) {
+    // FIXME(akoen): Current implementation assumes
+    // angle > 0 = right
+    // angle < = left
+
+    return (TRACK_WIDTH_mm*tanf(wheel_angle * ((float) M_PI / 180.0f))) / (2 * WHEELBASE_mm);
```

Comment:
General comment for this entire PR: Consider breaking calculations into multiple lines where it would improve readability. For ex. this would probably read better as:
```c
const float wheel_angle_rad = wheel_angle * ((float) M_PI / 180.0f));
return TRACK_WIDTH_MM * tanf(wheel_angle_rad) / (2 * WHEELBASE_MM);
```

Reply:
Also it's pretty common to implement trig functions via lookup table on embedded, idk if that's necessary here though

Reply:
Also I'd imagine the degree to rad conversion could be added to the `App_SharedMacros.h` file, seems useful to have in shared code